src/include/sof/common.h

```diff
@@ -14,14 +14,8 @@
 /* callers must check/use the return value */
 #define __must_check __attribute__((warn_unused_result))
 
-#ifdef __ZEPHYR__
-#include <zephyr/sys/util.h>
-#endif
-
 /* Align the number to the nearest alignment value */
-#ifndef IS_ALIGNED
 #define IS_ALIGNED(size, alignment) ((size) % (alignment) == 0)
-#endif
```

Comment:
if https://github.com/zephyrproject-rtos/zephyr/pull/67243 happens to be merged before this (yes, I know that it's currently blocked) and we move to a Zephyr version after it, this would break compilation. Why not just merge these two commits?

Reply:
we probably should hold of until we can get the Zephyr interface merged in. We have the SMP interface in flight (plus the regression), so the air is getting too thick with incompatible combinations.